### DISCUSSION TITLE
Minecraft crashes when going to the nether with shaders enabled.

### DISCUSSION THREAD

**Discussion by luke166** [IrisShaders/Iris#1201] [Q&A]
Hi, it seems that Minecraft crashes every time I attempt to enter the nether. I've found that disabling shaders prevents the crash, but it would be great if I didn't have to do that every single time. Any ideas?

I'm running version 1.18 with SEUS-Renewed shaders.

**Comment by BeansBeefBroccoli** [comment]
Please [open an issue](https://github.com/IrisShaders/Iris/issues/new?assignees=&labels=crash&template=crash_report.yml) instead.
